packages/backend/src/open_payments/account/routes.ts

```diff
@@ -36,8 +36,9 @@ export async function getAccount(
   const config = await deps.config
   ctx.body = {
     id: `${config.publicHost}/accounts/${encodeURIComponent(accountId)}`,
-    accountServicer: config.publicHost,
+    publicName: account.publicName,
     assetCode: account.asset.code,
-    assetScale: account.asset.scale
+    assetScale: account.asset.scale,
+    authServer: config.authServerGrantUrl
   }
 }
```

Comment:
Could you update the comment to point to [here](https://docs.openpayments.guide/reference/get-public-account). That's the place in the docs that I still need to update but that will include that endpoint.

Reply:
It doesn't show here. I mean the comment above the method.